docker/docker-compose.blockchain-link-test.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  electrum-regtest:
+    image: ghcr.io/vdovhanych/electrs_v2:latest
```

Comment:
Use the original image please `ghcr.io/vdovhanych/electrs:latest` it's now essentially the same as v2 but i will delete it as the ci builds only `electrs` one

Reply:
Great! done by 46f17bf4698252f75842912f0f513fec21757b78